package.json

```diff
@@ -24,7 +24,7 @@
     "url": "https://github.com/Microsoft/clarity-tools/issues"
   },
   "dependencies": {
-    "clarity-js": "^0.2.1",
+    "clarity-js": "^0.2.6",
```

Comment:
>"clarity-js": "^0.2.6", [](start = 4, length = 23)

Should there also be a change to the visualizer to handle Source.CSS ?

Reply:
No, since Source.Css isn't supported yet in the extension due to sandbox extension operates in.